scipy/special/add_newdocs.py

```diff
@@ -374,15 +374,15 @@ def add_newdoc(name, doc):
 
     Binomial distribution cumulative distribution function.
 
-    Sum of the terms 0 through `k` of the Binomial probability density.
+    Sum of the terms 0 through `floor(k)` of the Binomial probability density.
 
     .. math::
-        \mathrm{bdtr}(k, n, p) = \sum_{j=0}^k {{n}\choose{j}} p^j (1-p)^{n-j}
+        \mathrm{bdtr}(k, n, p) = \sum_{j=0}^floor(k) {{n}\choose{j}} p^j (1-p)^{n-j}
 
     Parameters
     ----------
     k : array_like
-        Number of successes (int).
+        Number of successes (double).
```

Comment:
Similarly the rounding needs to be mentioned.

Reply:
done. added a note that `k` is rounded down to nearest integer.